quasar-utils/javac-plugin/src/main/java/co/paralleluniverse/fibers/SuspendExecution.java

```diff
@@ -0,0 +1,3 @@
+package co.paralleluniverse.fibers;
+
+public class SuspendExecution extends Exception {}
```

Comment:
This is a Quasar class is a Quasar package. Why is it here? ("Split packages" being Very Naughty these days, etc.)

Reply:
I put it here to avoid having the plugin depending on quasar, but I actually only need the name of this class, which means I can easily remove it